src/main/resources/META-INF/plugin.xml

```diff
@@ -334,6 +334,9 @@
       <separator/>
       <!-- helm -->
       <action class="org.jboss.tools.intellij.openshift.actions.helm.OpenHelmChartsAction" id="org.jboss.tools.intellij.openshift.actions.helm.OpenHelmChartsAction" text="Open Helm Charts"/>
+      <action class="org.jboss.tools.intellij.openshift.actions.helm.AddHelmRepoAction" id="org.jboss.tools.intellij.openshift.actions.helm.AddHelmRepoAction" text="Add Repository"/>
+      <action class="org.jboss.tools.intellij.openshift.actions.helm.EditHelmRepoAction" id="org.jboss.tools.intellij.openshift.actions.helm.EditHelmRepoAction" text="Edit Repository"/>
+      <action class="org.jboss.tools.intellij.openshift.actions.helm.RemoveRepositoriesAction" id="org.jboss.tools.intellij.openshift.actions.helm.RemoveRepositoriesAction" text="Remove Repositories"/>
```

Comment:
worth a minor improvement, but since we can tweak the text on the action, can we use a single or plural text depending on selected elements ?

Reply:
@sbouchet: not sure if it really is worth it because with these updates the text in the xml becomes quasi meaningless which then isn't fully obvious at first sight. But I could do it if you really think that it's needed.